docker/rumble/scripts/python/rumble_client.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+"""
+Author:      Ceyhun Uzunoglu <ceyhunuzngl AT gmail dot com>
+Description: Client to run jsoniq queries in rumble server.
+"""
+
+# system modules
+import os
+import json
+import argparse
+
+# third party libs
+import requests
+
+
+class OptionParser:
+	def __init__(self):
```

Comment:
Again, the indentation here is non standard, you use now 7 white spaces, while it should be 4. Please fix the python code with pep8 guidelines.